src/content/operations/troubleshooting/_index.en.md

```diff
@@ -33,11 +33,55 @@ This section will contain information about common deployment issues you can run
 
 #### TBD
 
+-->
+
 ### Connectivity Issues
 
 Submariner deployment completed successfully but Services/Pods on one cluster are unable to connect to Services on another cluster. This can
-be due to multiple factors outlined in the following sections.
+be due to multiple factors outlined below.
+
+#### Check the Connection Statistics
+
+If you are unable to connect to a remote cluster, check its connection status in the Gateway resource.
+
+`kubectl describe Gateway -n submariner-operator`
+
+Sample output:
 
+```yaml
+   - endpoint:
+        backend: strongswan
```

Comment:
```suggestion
        backend: libreswan
```

Reply:
Let's try to use libreswan in our examples, now that strongswan is being deprecated.

Reply:
Done.